codec-http2/src/main/java/io/netty/handler/codec/http2/DelegatingDecompressorFrameListener.java

```diff
@@ -52,16 +52,30 @@ public class DelegatingDecompressorFrameListener extends Http2FrameListenerDecor
     private final boolean strict;
     private boolean flowControllerInitialized;
     private final Http2Connection.PropertyKey propertyKey;
+    private final int maxAllocation;
 
+    @Deprecated
     public DelegatingDecompressorFrameListener(Http2Connection connection, Http2FrameListener listener) {
-        this(connection, listener, true);
+        this(connection, listener, 0);
     }
 
     public DelegatingDecompressorFrameListener(Http2Connection connection, Http2FrameListener listener,
-                    boolean strict) {
+                                               int maxAllocation) {
+        this(connection, listener, true, maxAllocation);
+    }
+
+    @Deprecated
+    public DelegatingDecompressorFrameListener(Http2Connection connection, Http2FrameListener listener,
+                                               boolean strict) {
+        this(connection, listener, strict, 0);
+    }
+
+    public DelegatingDecompressorFrameListener(Http2Connection connection, Http2FrameListener listener,
```

Comment:
Consider adding javadoc for all constructors

Reply:
Added